ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/views/NoActiveUserManagementView.kt

```diff
@@ -56,31 +66,53 @@ private fun ContentUnavailableView(
     description: String?,
     modifier: Modifier = Modifier,
 ) {
-    Column(
-        modifier = modifier
-            .fillMaxWidth()
-            .padding(ContentUnavailableViewPadding),
-        horizontalAlignment = Alignment.CenterHorizontally,
+    Surface(
+        modifier = modifier,
+        shape = RoundedCornerShape(CustomerCenterConstants.Card.ROUNDED_CORNER_SIZE),
+        color = MaterialTheme.colorScheme.surface,
     ) {
-        Icon(
-            imageVector = Icons.Rounded.Warning,
-            contentDescription = null,
-            modifier = Modifier.size(ContentUnavailableIconSize),
-        )
-
-        Text(
-            text = title,
-            style = MaterialTheme.typography.headlineSmall,
-            modifier = Modifier.padding(top = ContentUnavailableViewPaddingTopTitle),
-        )
+        Column(
+            modifier = Modifier.fillMaxWidth()
+                .padding(all = 20.dp),
+            horizontalAlignment = Alignment.CenterHorizontally,
+        ) {
+            Icon(
+                imageVector = Icons.Rounded.Info,
+                contentDescription = null,
+                modifier = Modifier.size(ContentUnavailableIconSize),
+            )
 
-        description?.let {
             Text(
-                text = it,
-                style = MaterialTheme.typography.bodyLarge,
-                textAlign = TextAlign.Center,
-                modifier = Modifier.padding(top = ContentUnavailableViewPaddingTopDescription),
+                text = title,
+                style = MaterialTheme.typography.titleMedium,
+                modifier = Modifier.padding(vertical = ContentUnavailableViewPaddingTitle),
             )
+
+            description?.let {
+                Text(
+                    text = it,
+                    style = MaterialTheme.typography.bodyMedium,
+                    textAlign = TextAlign.Center,
+                )
+            }
         }
     }
 }
+
+@Preview
+@Composable
+private fun NoActiveUserManagementView_Preview() {
```

Comment:
I'm thinking it could be nice to add previews for dark mode as well, so we get snapshot tests automatically.

Reply:
Hmm weird... seems like the background is dark mode but it's not getting propagated to the view itself... Maybe it has something to do with the surface and the colors there not responding correctly in the previewer? Still I would expect that to happen also in the AS previewer 🤔. Might be worth contacting Emerge if it only reproduces there.